community_server/src/Controller/JsonRequestHandlerController.php

```diff
@@ -340,12 +340,13 @@ private function sendEMailTransactionFailed($transaction, $reason_type)
             return;
         }
         $transaction_body = $transaction->getTransactionBody();
-        $transaction_type_name = $transaction_body->getTransactionTypeName();
-        $senderUser = null;
-        if($transaction_type_name === 'transfer') {
-            $senderUser = $transaction_body->getSpecificTransaction()->getSenderUser();
-        } else if($transaction_type_name === 'creation') {
-            $senderUser = $transaction->getFirstSigningUser();
```

Comment:
I have refactored the code to have less repetitions.
Now  $senderUser = $transaction->getFirstSigningUser() is called everytime. 
Because every transaction have at least one user how signed it and usually it is the sender of the transaction.